lib/classes/PluginManager.js

```diff
@@ -372,24 +372,31 @@ class PluginManager {
         };
       }
       if (!_.isObject(options)) {
-        throw new Error(
+        throw new ServerlessError(
           `Custom variable resolver {${variablePrefix}: ${JSON.stringify(
             options
-          )}} defined by ${pluginName} isn't an object`
+          )}} defined by ${pluginName} isn't an object`,
+          'CUSTOM_LEGACY_VARIABLES_RESOLVER_INVALID_FORMAT'
         );
       } else if (!variablePrefix.match(/[0-9a-zA-Z_-]+/)) {
-        throw new Error(
-          `Custom variable resolver prefix ${variablePrefix} defined by ${pluginName} may only contain alphanumeric characters, hyphens or underscores.`
+        throw new ServerlessError(
+          `Custom variable resolver prefix ${variablePrefix} defined by ${pluginName} ` +
+            'may only contain alphanumeric characters, hyphens or underscores.',
+          'CUSTOM_LEGACY_VARIABLES_RESOLVER_INVALID_PREFIX'
         );
       }
       if (typeof options.resolver !== 'function') {
-        throw new Error(
-          `Custom variable resolver for ${variablePrefix} defined by ${pluginName} specifies a resolver that isn't a function: ${options.resolver}`
+        throw new ServerlessError(
+          `Custom variable resolver for ${variablePrefix} defined by ${pluginName} ` +
+            `specifies a resolver that isn't a function: ${options.resolver}`,
+          'CUSTOM_LEGACY_VARIABLES_RESOLVER_INVALID_FORMAT'
         );
       }
       if (options.isDisabledAtPrepopulation && typeof options.serviceName !== 'string') {
-        throw new Error(
-          `Custom variable resolver for ${variablePrefix} defined by ${pluginName} specifies isDisabledAtPrepopulation but doesn't provide a string for a name`
+        throw new ServerlessError(
+          `Custom variable resolver for ${variablePrefix} defined by ${pluginName} ` +
+            "specifies isDisabledAtPrepopulation but doesn't provide a string for a name",
+          'USTOM_LEGACY_VARIABLES_RESOLVER_INVALID_CONFIGURATION'
```

Comment:
There's `C` missing at the beginning of error code

Reply:
Good find! Fixed